src/main/webapp/i18n/de/plagiarism.json

```diff
@@ -9,6 +9,7 @@
             "warning": "Die Bestätigung/Ablehnung eines Plagiats geht beim Neuladen der Seite verloren.",
             "inspection": "Plagiatskontrolle",
             "plagiarism": "Plagiat",
+            "submissions": "Studierenden-Abgaben",
```

Comment:
Other occurrences call this "Einreichungen"

Reply:
Changed.